src/safe_cli/operators/safe_operator.py

```diff
@@ -182,6 +186,7 @@ def __init__(self, address: ChecksumAddress, node_url: str):
             True  # Require all signatures to be present to send a tx
         )
         self.hw_wallet_manager = get_hw_wallet_manager()
+        self.script_mode = script_mode  # Disable prompt dialogs
```

Comment:
I think `script_mode` can be confusing as it might suggest that we offering ways to load and scripts made specifically for the `safe-cli`. The way I see it is that this is to support calls from scripts where there won't be any user input. If that's the case maybe we can rename this to `no_input` what do you think?

This feature can be enabled via `--no-input` and would be disabled for REPL by default.

Reply:
Done [here](https://github.com/safe-global/safe-cli/pull/419/commits/1e1d3548bd4c283e2de19261d3936d82d86bf698). Do you see it necessary to add the `--no-input` option to all methods? It may not aport a lot, as internally it is easy to differentiate when the attended mode or unattended mode is used.

Reply:
> Done [here](https://github.com/safe-global/safe-cli/pull/419/commits/1e1d3548bd4c283e2de19261d3936d82d86bf698). Do you see it necessary to add the `--no-input` option to all methods? It may not aport a lot, as internally it is easy to differentiate when the attended mode or unattended mode is used.

I'd start with adding to the methods that require some input and try to find a way to have it for all the methods. The main reason being that it's an option that makes sense for any command even if there's no user prompt for some of them. But this makes it learning the CLI easier as the the user wouldn't need to know where `--no-input` is valid or not.

If there's a way in Typer to have "common" configs then we could use that.